.circleci/ci-packages.txt

```diff
@@ -139,7 +139,7 @@ perl-Module-Pluggable-5.2
 perl-Module-Runtime-0.016
 perl-Module-Runtime-Conflicts-0.003
 perl-Mojo-IOLoop-ReadWriteProcess-0.28
-perl-Mojolicious-8.71
+perl-Mojolicious-8.72
```

Comment:
Why was the Mojolicious version update needed?

Reply:
Just out of curiosity... not that i would ever complain about a Mojolicious version update. 😁

Reply:
Because the previous version was already unavailable :)